projects/gui/src/gamedatabasedlg.cpp

```diff
@@ -302,6 +306,18 @@ GameDatabaseDialog::GameDatabaseDialog(GameDatabaseManager* dbManager, QWidget*
 	connect(ui->m_createOpeningBookBtn, SIGNAL(clicked(bool)), this,
 		SLOT(createOpeningBook()));
 
+	connect(ui->m_copyGameBtn, SIGNAL(pressed()), this, SLOT(copyGame()));
+	connect(ui->m_copyGameBtn, &QPushButton::released, this, [=]()
+	{
+		ui->m_copyGameBtn->setText(tr("Copy game"));
```

Comment:
Maybe it would be more consistent to use "Copy PGN" and "Copy FEN" or "Copy game" and "Copy position". I like the first option better because then the user will know the format of the game notation and we use "PGN" in a lot of other places too.

Reply:
changed accordingly